contracts/truefi2/TrueCreditAgency.sol

```diff
@@ -300,7 +300,7 @@ contract TrueCreditAgency is UpgradeableClaimable, ITrueCreditAgency {
      * @param borrower Borrower to get borrow limit for
      * @return borrow limit for `borrower` in `pool`
      */
-    function borrowLimit(ITrueFiPool2 pool, address borrower) public view returns (uint256) {
+    function borrowLimit(ITrueFiPool2 pool, address borrower) public override view returns (uint256) {
```

Comment:
I agree, LoC's `borrowLimit()` shouldn't be also used for FTL. We have common `borrowLimit()` in TRA, and honestly, since both LoC and FTL would use it, I think TRA could use a better naming.

> Perhaps both of them could pass in their own `totalBorrowed(borrower, decimals)` values as arguments to `borrowLimit()`?

I believe this is the way to go.